jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -471,6 +476,45 @@ private static Path getCheckedCacheDirectory(String property, Path defaultPath)
     return defaultPath;
   }
 
+  /**
+   * Checks whether or not the known default Java 8 distroless base image is being used. Checking
+   * against only images known to Java 8, the method may to return {@code false} for Java 8
+   * distroless unknown to it.
+   *
+   * @param baseImage the configured base image
+   * @return {@code true} if the base image is equal to one of the known Java 8 distroless images,
+   *     else {@code false}
+   */
+  private static boolean isKnownDistrolessJava8Image(String baseImage) {
+    // TODO: drop "latest" and the likes once it no longer points to Java 8.
+    return baseImage.equals("gcr.io/distroless/java")
+        || baseImage.equals("gcr.io/distroless/java:latest")
+        || baseImage.equals("gcr.io/distroless/java:debug")
+        || baseImage.equals("gcr.io/distroless/java:8")
+        || baseImage.equals("gcr.io/distroless/java:8-debug")
+        || baseImage.equals("gcr.io/distroless/java/jetty")
+        || baseImage.equals("gcr.io/distroless/java/jetty:debug")
+        || baseImage.equals("gcr.io/distroless/java/jetty:java8")
+        || baseImage.equals("gcr.io/distroless/java/jetty:java8-debug");
+  }
+
+  /**
+   * Checks whether or not the known default Java 11 distroless base image is being used. Checking
```

Comment:
_Checks if the base image is a known Java 11 distroless image._?

Reply:
Done. Also noticed the image doesn't have to be a base image. Fixed other minor things too.